### ISSUE TITLE
No way to specify ffprobe path

### ISSUE THREAD

**Issue by juanique** [u2takey/ffmpeg-go#117]
This code https://github.com/u2takey/ffmpeg-go/blob/6649d14c4aeb35093a062d4660a55507198cd03e/probe.go#L35 assumes that `ffprobe` is on the system PATH. There should be a way to specify the path of the executable to run.

**Comment by nzoschke** [comment]
Yes this follows from the ability to set a different ffmpeg path via `ffmpeg.SetFfmpegPath(name)`.

It is important to be able to specify non-system binaries for both `ffmpeg` and `ffprobe`
